CONTRIBUTING.md

```diff
@@ -34,6 +34,13 @@ In general we'd like to ask you to please target the master branch with your pul
 We recommend creating a new branch for each feature task/bug since it makes easier to edit
 commits in future if needed.
 
+After submitting a pull request preview your changes by going through the checks section in
+last of the pull request page and click `Details` of the `netlify/cranky-borg-621833/deploy-preview` check.
```

Comment:
There is a separate `Checks` tab. We can talk about that instead of this approach.

Reply:
Yes wrt to the debugging and review going to the checks tab and previewing every page changed is best, also index link same so anyone can also preview any other page also, so I am also in favour of going through the checks tab and previewing every tab

Should I change the screenshot and update the instructions?

Reply:
Yes, we can update this text to tell people to click that tab and access the changed files.